src/main/kotlin/org/datepollsystems/waiterrobot/mediator/printer/AbstractLocalPrinter.kt

```diff
@@ -0,0 +1,35 @@
+package org.datepollsystems.waiterrobot.mediator.printer
+
+import org.apache.pdfbox.Loader
+import org.apache.pdfbox.pdmodel.PDDocument
+import java.util.*
+
+/**
+ * Manages a local Printer
+ *
+ * for an example see [org.datepollsystems.waiterrobot.mediator.printer.LocalPrinterTest.testPrint]
+ *
+ * @author Fabian Schedler
+ * @see org.datepollsystems.waiterrobot.mediator.printer.LocalPrinterTest.testPrint
+ */
+abstract class AbstractLocalPrinter : LocalPrinterInfo {
+
+    fun printPdf(pdfId: String, base64data: String) {
+        val decoded = Base64.getDecoder().decode(base64data)
+        printPdf(pdfId, Loader.loadPDF(decoded))
+    }
```

Comment:
Just for aesthetics, this could be a one liner :)
```suggestion
    fun printPdf(pdfId: String, base64data: String) = printPdf(
      pdfId, Loader.loadPDF(Base64.getDecoder().decode(base64data))
    )
```

Reply:
Not really a fan of nesting multiple function calls in parameters as it is harder to read.